src/components/base/JThumbnail/JThumbnail.js

```diff
@@ -1,36 +1,45 @@
 // @flow
 
 import React, { PureComponent } from 'react'
+import classNames from 'classnames'
 
 import { JText, JIcon } from 'components/base'
 
-type JThumbnailImage = 'key' | 'man' | 'cloud' | 'auth-question' | 'auth-cross'
+type JThumbnailImage = 'key' | 'man' | 'cloud' | 'auth-question' | 'auth-cross' | 'screen-search'
 
 type Props = {|
   +title: string,
   +description: string | Array<string>,
   +image: JThumbnailImage,
-  +color: 'white' | 'gray' | 'blue',
+  +iconSize: 'large' | 'xlarge',
+  +color: 'white' | 'gray' | 'blue' | 'red',
+  isTransparent: boolean,
 |}
 
 class JThumbnail extends PureComponent<Props, *> {
   static defaultProps = {
     title: '',
+    iconSize: 'xlarge',
     color: 'white',
+    isTransparent: false,
     description: '',
   }
 
   render() {
     const {
       image,
+      iconSize,
       color,
+      isTransparent,
       title,
       description,
     } = this.props
 
     return (
       <div className='j-thumbnail' >
-        <JIcon size='xlarge' name={image} />
+        <div className={classNames('icon', isTransparent && '-transparent')}>
```

Comment:
please stop use `component` props as children elements variants.
you need to use isTransparent variant for j-thumbnail